src/main/java/com/monumental/listeners/MonumentListener.java

```diff
@@ -0,0 +1,52 @@
+package com.monumental.listeners;
+
+import com.monumental.models.Monument;
+import com.monumental.services.MonumentService;
+import org.hibernate.event.spi.PreInsertEvent;
+import org.hibernate.event.spi.PreInsertEventListener;
+import org.hibernate.event.spi.PreUpdateEvent;
+import org.hibernate.event.spi.PreUpdateEventListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
`@Component` is used to make Spring find the class and make it a `Bean`, meaning other Spring classes can `@Autowired` it. It's a way to connect this Hibernate stuff to the Spring stuff